htdocs/takepos/index.php

```diff
@@ -329,8 +331,15 @@ function LoadProducts(position, issubcat) {
 	if (maxproduct >= 1) {
 		limit = maxproduct-1;
 	}
+
+	// Get socid
+	let socid = jQuery('#thirdpartyid').val();
+	if ((socid === undefined || socid === "") && parseInt("<?php echo $socid ?>") > 0) {
+		socid = parseInt("<?= $socid ?>");
```

Comment:
Could you put `<?php echo dol_escape_js($socid); ?>` ?
@eldy is it the good way to do ?